<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form_SceneManager</class>
    <widget class="QWidget" name="Form_SceneManager">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QTreeWidget" name="treeWidget">
                    <property name="columnCount">
                        <number>2</number>
                    </property>
                    <column>
                        <property name="text">
                            <string notr="true">1</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string notr="true">2</string>
                        </property>
                    </column>
                </widget>
            </item>
            <item>
                <layout class="QGridLayout" name="gridLayout">
                    <item row="3" column="0">
                        <widget class="QCheckBox" name="checkBox_ShowType">
                            <property name="text">
                                <string>Show Type</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QPushButton" name="pushButton_AddNode">
                            <property name="text">
                                <string>Add Node</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QPushButton" name="pushButton_RemoveNode">
                            <property name="text">
                                <string>Remove Selected Nodes</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
